code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -561,16 +561,11 @@
 				continue
 			step_towards(alive_mob, loc)
 
-	//Gases can be removed from the moderator internal by using the interface. Helium and antinoblium inside the fusion mix will get always removed at a fixed rate
-	if(waste_remove && power_level <= 5)
-		var/filtering = TRUE
-		if(!ispath(filter_type))
-			if(filter_type)
-				filter_type = gas_id2path(filter_type)
-			else
-				filtering = FALSE
-		if(filtering && moderator_internal.gases[filter_type])
-			var/datum/gas_mixture/removed = moderator_internal.remove_specific(filter_type, 100 * delta_time)
+	//Gases can be removed from the moderator internal by using the interface.
+	if(waste_remove)
```

Comment:
Was letting people remove at fusion level 6 intentional?

Reply:
ah yes, i forgot to add it to the CL, thanks for the catch!